src/alg_utils.jl

```diff
@@ -171,7 +171,17 @@ function DiffEqBase.prepare_alg(alg::Union{OrdinaryDiffEqAdaptiveImplicitAlgorit
     if typeof(alg) <: OrdinaryDiffEqImplicitExtrapolationAlgorithm
       return alg # remake fails, should get fixed
     else
-      remake(alg,chunk_size=Val{ForwardDiff.pickchunksize(x)}())
+      chunk_size = ForwardDiff.pickchunksize(x)
+      if chunk_size > 8
+        cs = Val{8}()
+        remake(alg,chunk_size=cs)
+      elseif chunk_size > 4
+        cs = Val{4}()
+        remake(alg,chunk_size=cs)
+      else
```

Comment:
Is it worth adding `2` and `3`?
I'm worried missing these could lead to too large a runtime regression in some examples.

Of course, that'd cost some compile time.